src/adapters/EasyRtcAdapter.js

```diff
@@ -246,14 +311,17 @@ class EasyRtcAdapter extends NoOpAdapter {
 
     this.easyrtc.setStreamAcceptor(this.setMediaStream.bind(this));
 
-    this.easyrtc.setOnStreamClosed(function(easyrtcid) {
-      delete that.mediaStreams[easyrtcid];
+    this.easyrtc.setOnStreamClosed(function(clientId, stream, streamName) {
+      delete this.mediaStreams[clientId][streamName];
     });
 
     if (that.easyrtc.audioEnabled || that.easyrtc.videoEnabled) {
-      this.easyrtc.initMediaSource(
+      navigator.mediaDevices.getUserMedia({
+        video: that.easyrtc.videoEnabled,
+        audio: that.easyrtc.audioEnabled
+      }).then(
```

Comment:
This is what `this.easyrtc.initMediaSource` was doing?
Maybe not in the PR, we should be able to specify options here 
`audio: {echoCancellation: true, noiseSuppression: true, autoGainControl: true}` instead of `audio: true`
and for example
`video: {mediaSource: "camera", width: { max: 1280, ideal: 640 }, height: { ideal: 360 }, frameRate: 30}` instead of `video: true`

Reply:
I created the issue #300 to track this.